content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -41,32 +48,38 @@ $ helm upgrade istio install/kubernetes/helm/istio --namespace istio-system
 
 You can also use Kubernetes’ rolling update mechanism to upgrade the control plane components. This is suitable for cases when Istio hasn't been installed using Helm.
 
-First, generate the desired Istio control plane yaml file, e.g.
+1. Upgrade all the Istio's CRDs via `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kube api-server:
```

Comment:
To make the actions the reader needs to take clearer, I suggest:
```suggestion
1. Use `kubectl apply` to upgrade all the Istio's CRDs.  Wait a few seconds for the Kubernetes API server to receive the upgraded CRDs:
```

Reply:
done